Services/Common/OJS.Workers/OJS.Workers.Common/IExecutionContext.cs

```diff
@@ -12,6 +12,8 @@ public interface IExecutionContext<TInput>
 
         byte[] FileContent { get; set; }
 
+        public byte[] AdditionalFiles { get; set; }
```

Comment:
The `additional files` for the execution context are mapped from an `OjsSubmission<TInput>`, whose `nullable types` are disabled, even if we make this property `byte[]?`, it would be mapped to an empty array and not to `null`. A similar approach is used with the `file content`.